.mocharc.yaml

```diff
@@ -0,0 +1,6 @@
+require: 

```

Comment:
use a `.js` file instead of yaml for .mocharc
as it would allow better re-use / combining

Reply:
this is .js now